roles/ceph-common/defaults/main.yml

```diff
@@ -277,7 +277,6 @@ mds_max_mds: 3
 ## Rados Gateway options
 #
 #radosgw_dns_name: your.subdomain.tld # subdomains used by radosgw. See http://ceph.com/docs/master/radosgw/config/#enabling-subdomain-s3-calls
-radosgw_frontend: civetweb # supported options are 'apache' or 'civetweb', also edit roles/ceph-rgw/defaults/main.yml
```

Comment:
Can we actually keep this and do some refactor?
I was thinking to use a variable to bring more flexibility to users, so we could have something like this:

`rgw frontends: "civetweb port={{ radosgw_civetweb_bind_ip }}:{{ radosgw_civetweb_port }} num_threads={{ radosgw_civetweb_num_threads }}"`

Potentially users can add more civetweb options and also decide to only specify a port and not a bind_ip.
So for we need to keep the existing variables:

* `radosgw_civetweb_bind_ip``
* `radosgw_civetweb_port``
* `radosgw_civetweb_num_threads``

For backward compatibility.

Of course we can address that into a new PR but I just don't want to delete and re-introduce something.

What do you guys think?